systers_portal/templates/base.html

```diff
@@ -209,6 +204,7 @@
 <script type="text/javascript" src="{% static 'js/community-search.js' %}"></script>
 <script src="{% static 'js/choose_profile_pic.js' %}"></script>
 <script src="{% static 'js/search_menu.js' %}"></script>
+<script src="{% static 'js/main.js' %}"></script>
```

Comment:
What was the need for main.js?

Reply:
Ohk I had removed the file but this was left here itself. Needs to be removed

Reply:
Removed!